src/roadmap/Roadmap.test.js

```diff
@@ -0,0 +1,32 @@
+import { render, screen } from '@testing-library/react';
+import { Roadmap } from './Roadmap';
+import '@testing-library/jest-dom/extend-expect';
+
+describe('roadmap initial state', () => {
+  test('Roadmap has title called Roadmap', () => {
+    render(<Roadmap />);
+    const roadmapTitle = screen.getByRole('heading', { name: 'Roadmap' });
+    expect(roadmapTitle).toHaveTextContent('Roadmap');
+  });
+  test.skip('Roadmap should have a button called View', () => {
+    render(<Roadmap />);
+    const roadMapButton = screen.getByRole('button', { name: 'View' });
+    expect(roadMapButton).toHaveTextContent('View');
+  });
+  test('Roadmap should have a status called planned with a value of 4', () => {
+    render(<Roadmap />);
+    const plannedLabel = screen.getByTestId('planned');
+    expect(plannedLabel).toHaveTextContent('planned');
```

Comment:
Your test name suggests that you're testing the value. Do you want to update that?

Reply:
Updated it to be more generic